### PR TITLE
linux: Enable usb wifi hotspot on wp-04

### PR DESCRIPTION
**Pull request by legendlee1314**

### CODE REVIEW

File: linux/config/dnsmasq.conf [hunk 1]
```diff
@@ -646,7 +646,7 @@
 #dhcp-range=192.168.42.50,192.168.42.200,12h
 
 
-interface=wlan0
+interface=wlan1
 bind-interfaces #这个是只监听wlan0，没有之会检测所有卡
```
Comment (gyagp) [inline linux/config/dnsmasq.conf]: Change the comment here

File: linux/ap.sh [hunk 2]
```diff
@@ -28,7 +28,7 @@ iptables -t nat -A POSTROUTING -s 192.168.0.0/8 -o $EXTERNAL -j MASQUERADE
 iptables -A FORWARD -s 192.168.0.0/8 -o $EXTERNAL -j ACCEPT
 iptables -A FORWARD -d 192.168.0.0/8 -m conntrack --ctstate ESTABLISHED,RELATED -i $EXTERNAL -j ACCEPT
 
-ifconfig wlan0 192.168.0.1
+ifconfig wlan1 192.168.0.1
```
Comment (gyagp) [inline linux/ap.sh]: Get wlan* from real machine

File: linux/config/hostapd/hostapd.conf [hunk 3]
```diff
@@ -52,6 +53,7 @@ eap_server=0
 own_ip_addr=127.0.0.1
 
 #WPA/WPA2密码在这，我也是不需要
-wpa=1
-wpa_psk=0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
-wpa_passphrase=perf12345
\ No newline at end of file
+wpa=2
+wpa_key_mgmt=WPA-PSK
+rsn_pairwise=CCMP
```
Comment (gyagp) [inline linux/config/hostapd/hostapd.conf]: Try to make minimal changes

File: linux/config/hostapd/hostapd.conf [hunk 4]
```diff
@@ -1,5 +1,6 @@
-interface=wlan0
-driver=nl80211
+interface=wlan1
+#driver=nl80211
```
Comment (gyagp) [inline linux/config/hostapd/hostapd.conf]: Remove this line

**Comment by gyagp** [comment]
Use <machine_name>-xxx.conf for machine specific configurations.

**Comment by gyagp** [comment]
Please write in detail how to set up ap on the machine, and backup files in an appropriate way.

**Comment by legendlee1314** [comment]
@gyagp , please help review again.

**Comment by legendlee1314** [comment]
@gyagp , please review

**Pull request merged**